src/components/BasicMap/BasicMap.js

```diff
@@ -225,16 +226,22 @@ class BasicMap extends Component {
       this.map.getView().setResolution(resolution);
     }
 
-    if (viewOptions && prevProps.viewOptions.extent !== viewOptions.extent) {
-      // Re-create a view, ol doesn't provide any method to setExtent of view.
-      this.map.setView(
-        new View({
-          ...viewOptions,
-          ...{ center },
-          ...{ resolution },
-          ...{ extent: viewOptions.extent },
-        }),
-      );
+    if (viewOptions && viewOptions.extent) {
+      if (
+        !prevProps.viewOptions.extent ||
+        (prevProps.viewOptions.extent &&
+          !equals(prevProps.viewOptions.extent, viewOptions.extent))
+      ) {
```

Comment:
In one if :

if (viewOptions && viewOptions.extent &&
    (!prevProps.viewOptions.extent ||
￼	        (prevProps.viewOptions.extent &&
￼	          !equals(prevProps.viewOptions.extent, viewOptions.extent))